packages/@react-native-windows/tester/src/js/utils/RNTesterList.windows.js

```diff
@@ -26,9 +26,24 @@ const ComponentExamples: Array<RNTesterExample> = [
     module: require('../examples/FlatList/FlatListExample'),
   },
   {
-    key: 'DatePickerExample',
-    category: 'UI',
-    module: require('../examples-win/DatePicker/DatePickerExample'),
+    key: 'FlatList-withSeparators',
+    module: require('../examples/FlatList/FlatList-withSeparators'),
+    category: 'ListView',
+  },
+  {
+    key: 'FlatList-onViewableItemsChanged',
+    module: require('../examples/FlatList/FlatList-onViewableItemsChanged'),
+    category: 'ListView',
+  },
+  {
+    key: 'FlatList-onEndReached',
+    module: require('../examples/FlatList/FlatList-onEndReached'),
+    category: 'ListView',
+  },
+  {
+    key: 'ImageExample',
+    category: 'Basic',
+    module: require('../examples/Image/ImageExample'),
   },
```

Comment:
Why is this removed? It looks like this was a custom windows page.

Reply:
Not removed - I moved it 5 lines above to align with core order and stop getting conflicts for this example.

Reply:
I still don't see DatePickerExample anywhere

Reply:
The comment is on the ImageExample line. For DatePicker,see Alex's comment and my response